app/src/main/java/org/fossasia/susi/ai/activities/ForgotPasswordActivity.java

```diff
@@ -95,7 +95,7 @@ public void signUp() {
         if(personalServer.isChecked()) {
             if(!CredentialHelper.checkIfEmpty(url,this) && CredentialHelper.isURLValid(url,this)) {
                 if (CredentialHelper.getValidURL(url,this) != null) {
-                    PrefManager.putBoolean("is_susi_server_selected", false);
+                    PrefManager.putBoolean(getApplicationContext().getString(R.string.action_select_server), false);
```

Comment:
@madhavrathi Sorry, I didn't realize it earlier but we do have a class named Constant for constant strings. https://github.com/fossasia/susi_android/blob/development/app/src/main/java/org/fossasia/susi/ai/helper/Constant.java . Please add `public static final String SUSI_SERVER="is_susi_server_selected";` in it and use it as `Constant.SUSI_SERVER` .